sdk/ml/azure-ai-ml/mypy.ini

```diff
@@ -0,0 +1,12 @@
+[mypy]
```

Comment:
Out of curiosity, why couldn't these be configured in `pyproject.toml`?

Reply:
The engineering system check guide for this repo [here](https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/static_type_checking.md#run-mypy) says to use a mypy.ini file. However, I'm finding that doesn't work, so I'm going to go back to using the pyproject.toml where azure-ai-ml sets all of our other configuration.